management/src/errors.ts

```diff
@@ -57,6 +57,15 @@ class IdentityNotFoundError extends Error {
   readonly message = "Identity Not Found";
 }
 
+class UnhandledIdentityType extends Error {
+  readonly message: string;
+
+  constructor(message: string) {
+    super();
+    this.message = message;
```

Comment:
```suggestion
    super(message);
```
Does this work? I think it does but not sure

Reply:
It does yes, we had a chat about this back then. I will change them 👍.